desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java

```diff
@@ -806,8 +811,10 @@ private void activateDisplayPreferences() {
 
         // We use opposite property (useStandbyMode) in preferences to have the default value (false) set as we want it,
         // so users who update gets set avoidStandbyMode=true (useStandbyMode=false)
-        avoidStandbyMode.setSelected(!preferences.isUseStandbyMode());
-        avoidStandbyMode.setOnAction(e -> preferences.setUseStandbyMode(!avoidStandbyMode.isSelected()));
+        if (displayStandbyModeFeature) {
+            avoidStandbyMode.setSelected(!preferences.isUseStandbyMode());
+            avoidStandbyMode.setOnAction(e -> preferences.setUseStandbyMode(!avoidStandbyMode.isSelected()));
+        }
```

Comment:
```suggestion
        } else {
            preferences.setUseStandbyMode(false)
        }
```

I agree with @ripcurlx, thus, the change suggestion. I believe that is a good place to put it. Instead of attaching a switch, we set it to false.